### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30016]
#### VFS team name
eBenefits

#### VFS product name
Dependency Verification

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
Add or remove a dependent on your VA disability benefits confirmation page should have a second review to clean up design and code to meet VA design system standards.

### Link, screenshot or steps to recreate
<img width="1384" alt="confirmation-page-markup" src="https://user-images.githubusercontent.com/14154792/133645890-79dcaa7c-fe92-4983-8455-6e8d4163e2e8.png">

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

---

### Platform Recommendation
Consider using more semantically appropriate HTML elements and following VA design system standards. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved
